src/modules/dracut/dracut.conf

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: no
+# SPDX-License-Identifier: CC0-1.0
+#
+# Run dracut(8) with an optional kernel name
+---
+# Dracut defaults to setting initramfs-<kernel-version>.img
+# If you want to specify another filename for the resulting image,
+# set a custom kernel name, including the path
+#
+# kernelName: /boot/initramfs-linux.img
```

Comment:
This should probably be uncommented, so that there is a specific thing to refer to. And since there is a config file, it should get a schema file (as @dalto8 says). It's eight lines or so.

Reply:
Default for dracut is to use initramfs-<kernel-version>.img, distributions that have been using this module for the last 8 years should not have to touch a (new) .conf to keep the default behavior.